test_conformance/api/test_api_min_max.cpp

```diff
@@ -1946,64 +1946,90 @@ int test_min_max_device_version(cl_device_id deviceID, cl_context context, cl_co
         return -1;
     }
 
```

Comment:
Please replace all the version parsing code above with `get_device_cl_version`.

Reply:
Thanks for your feedback. I have update te parsing code, I wasn't sure about the checking of the `OPENCL_1_0_DEVICE` and `OPENCL_1_1_DEVICE`, I assume we still want to keep these checks so I've left them in.

Reply:
I'd get rid of these, I think they were used to adapt the CTS behaviour to 1.1 and 1.0 implementations. We're now using queries for this.

Reply:
Thanks, okay that makes sense, I have removed them and reduced the scope of the loop counters and the `error` variable.